std-bits/table/src/main/java/org/enso/table/read/DelimitedReader.java

```diff
@@ -202,14 +200,30 @@ private void reportProblem(Problem problem) {
     }
   }
 
+  private Row internalReadNextRow() {
```

Comment:
`internal` prefix in the name of the method is a bit of an unusual convention. Isn't this obvious from the `private` modifier of the method?

Reply:
Indeed, it's just that I also have a `readNextRow` which is to be used normally. But I think you are right, I will try to rename these two somehow to more closely reflect their semantics and not their visibility.